jenkinsapi_tests/systests/test_crumbs_requester.py

```diff
@@ -22,89 +17,82 @@
 ENABLE_CRUMBS_CONFIG = {
     '': '0',
     'markupFormatter': {
-      'stapler-class': 'hudson.markup.EscapedMarkupFormatter',
-      '$class': 'hudson.markup.EscapedMarkupFormatter'
+        'stapler-class': 'hudson.markup.EscapedMarkupFormatter',
+        '$class': 'hudson.markup.EscapedMarkupFormatter'
     },
     'hudson-security-csrf-GlobalCrumbIssuerConfiguration': {
-      'csrf': {
-        'issuer': {
-          'value': '0',
-          'stapler-class': 'hudson.security.csrf.DefaultCrumbIssuer',
-          '$class': 'hudson.security.csrf.DefaultCrumbIssuer',
-          'excludeClientIPFromCrumb': False
+        'csrf': {
+            'issuer': {
+                'value': '0',
+                'stapler-class': 'hudson.security.csrf.DefaultCrumbIssuer',
+                '$class': 'hudson.security.csrf.DefaultCrumbIssuer',
+                'excludeClientIPFromCrumb': False
+            }
         }
-      }
     },
     'jenkins-model-DownloadSettings': {
-      'useBrowser': False
+        'useBrowser': False
     },
     'core:apply': '',
 }
 DISABLE_CRUMBS_CONFIG = {
     '': '0',
     'markupFormatter': {
-      'stapler-class': 'hudson.markup.EscapedMarkupFormatter',
-      '$class': 'hudson.markup.EscapedMarkupFormatter'
+        'stapler-class': 'hudson.markup.EscapedMarkupFormatter',
+        '$class': 'hudson.markup.EscapedMarkupFormatter'
     },
     'hudson-security-csrf-GlobalCrumbIssuerConfiguration': {},
     'jenkins-model-DownloadSettings': {
-      'useBrowser': False
+        'useBrowser': False
     },
     'core:apply': ''
-  }
+}
+
 
+@pytest.fixture(scope='function')
+def crumbed_jenkins(jenkins):
+    jenkins.requester.post_and_confirm_status(
+        jenkins.baseurl + '/configureSecurity/configure',
```

Comment:
you can use `urljoin` to be sure that result will not contain extra slashes (e.g. if baseurl ends with one)